docs/UserGuide.md

```diff
@@ -116,7 +116,9 @@ First, we need to make sure your computer has [Java](#technical-terms) 17 instal
 
 ### Step 3: Try Your First Commands
 
-Now that PROperty is running, let's try using the different commands available. All you have to do is to type the command in the command box and press Enter to execute it. 
+Now that PROperty is running, let's try using the different commands available. 
+We recommend you use a window size of at least 1920 x 1080 pixels to ensure all elements are displayed correctly.
+All you have to do is to type the command in the command box and press Enter to execute it. 
```

Comment:
Maybe can improve the spacing on this one, or even convert the window recommendation into a banner (the one with the background colour + icon like a tip) to improve the design.

Reply:
Okie I changed it! Let me know it it's okay